health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -172,31 +228,31 @@
                 // Initialize counter for this nesting level if it doesn't exist
                 const nestingKey = prefix || 'root';
                 if (!activeCounters[nestingKey]) {
                     activeCounters[nestingKey] = 0;
                 }
-                
+
                 // Only increment counter for active questions
                 if (question.isActive) {
                     activeCounters[nestingKey] += 1;
                     // Build the counter string (e.g., "2.1" or "2.1.3")
                     parentCounter = parentCounter + '.' + activeCounters[nestingKey];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)